src/components/toast-notification/ToastNotification.js

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types'
 import React, { PureComponent } from 'react'
-import { NotificationStack } from '@ubergrape/react-notification'
```

Comment:
@AleshaOleg we can replace it with `react-notification` since @kof back then made a PR and it was merged an released